apps/group/app/src/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
+import registerServiceWorker from './registerServiceWorker'
+
+ReactDOM.render(<App />, document.getElementById('root'))
+registerServiceWorker()
```

Comment:
I don't think this is possible, since it runs in a sandboxed iframe.

Reply:
Yes, removing this (it’s generated by create-react-app).